### PR TITLE
[Client Hints] AD review comments

### PR DESCRIPTION
**Pull request by yoavweiss**

### CODE REVIEW

File: draft-ietf-httpbis-client-hints.md [hunk 1]
```diff
@@ -162,55 +163,59 @@ The opt-in MUST be ignored if delivered over non-secure transport (using a schem
...
 
 ## Information Exposure
-Request header fields used in features relying on this document expose information about the user's environment to enable proactive content negotiation. Such information might reveal new information about the user and implementers ought to consider the following considerations, recommendations, and best practices.
 
-The underlying assumption is that exposing information about the user as a request header is equivalent (from a security perspective) to exposing this information by other means. (For example, if the request's origin can access that information using JavsScript APIs, and transmit it to its servers).
+Request header fields used in features relying on this document expose information about the user's environment to enable privacy-preserving proactive content negotiation, and avoid exposing passive fingerprinting vectors.
+However, implementers need to bear in mind that in the worst case, uncontrolled and unmonitored active fingerprinting is not better than passive fingerprinting. In order to provide user privacy benefits, user agents need to apply further policies that prevent abuse of the information exposed by features using Client Hints.
```
Comment (kaduk) [inline draft-ietf-httpbis-client-hints.md:193]: Thanks, this is a good way to frame the subsequent discussion to keep the relevant topics prominent.

File: draft-ietf-httpbis-client-hints.md [hunk 2]
```diff
@@ -121,17 +122,17 @@ A Client Hint request header field is a HTTP header field that is used by HTTP u
 ## Sending Client Hints
 
 User agents choose what Client Hints to send in a request based on their default settings, user configuration, and server preferences expressed in `Accept-CH`.
-The user agent and server can use an opt-in mechanism outlined below to negotiate which header fields need to be sent to allow for efficient content adaption, and optionally use additional mechanisms to negotiate delegation policies that control access of third parties to those same header fields.
-Without such an opt-in, user agents SHOULD NOT send high-entropy hints, but MAY send low-entropy ones {{CLIENT-HINTS-INFRASTRUCTURE}}.
+The user agent and server can use an opt-in mechanism outlined below to negotiate which header fields need to be sent to allow for efficient content adaption, and optionally use additional mechanisms (e.g., as outlined in {{CLIENT-HINTS-INFRASTRUCTURE}}) to negotiate delegation policies that control access of third parties to those same header fields.
+Without such an opt-in, user agents SHOULD NOT send hints that are not listed in the low-entropy hint table at {{CLIENT-HINTS-INFRASTRUCTURE}}.
```
[the remark this answers is not in the document]
Comment (yoavweiss) [reply draft-ietf-httpbis-client-hints.md:126]: Done

File: draft-ietf-httpbis-client-hints.md (hunk 1 above)
Comment (yoavweiss) [reply draft-ietf-httpbis-client-hints.md:193]: Thanks! :)

**Comment by mostafapes** [comment]
ipad air

**Comment by yoavweiss** [comment]
@mnot - PTAL

**Comment by yoavweiss** [comment]
> Magnus' comment makes me wonder if we should mention that CH's that want to use Accept-CH need to have a field name that matches sf-token; possibly in the Deployment and Security Risks section.

Added "A Client Hints request header negotiated using the Accept-CH opt-in mechanism MUST have a field name that matches sf-token (Section 3.3.4 of {{!I-D.ietf-httpbis-header-structure}})."

**Pull request merged**